### ISSUE TITLE
https://github.com/dotnet/VisualStudioCodeClient/issues/1660

### ISSUE THREAD

**Issue by Latha2564** [dotnet/machinelearning-modelbuilder#1730]
GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code Client;#Focus Order;#Keyboard;#WCAG2.4.3;#Win32;#Win10;#AINotInScope;VisualStudioCode-Mac-Aug2021

### Environment Details:​
Application Name: Visual Studio Code
Standard Violated: Usability
### Repro Steps:​
1.Open Visual Studio Code editor
2.Tab till Welcome tab and press enter
3.Tab till "Show All" link and press Enter.
4.Tab till "Search field" and populate some random data in Search field.
5.Verify whether the status message will appear on the screen and Voiceover is reading the status message or not.
### Actual Result:
When we populate some invalid data in the Search field "No status message" will appear on the Screen.
Note:
The similar  issue appears in Keyboard Shortcuts page.

### Expected Result: ​
When we populate some invalid data the Status message like "No data available" or 'No results found" should appear on the Screen and also the Screen reader should read the status message.

### User Impact:
Screen Reader users will find difficult in finding the Search results if no status message will appear on the screen and also the status message not read by the Screen Reader.

**Comment by Latha2564** [comment]
https://user-images.githubusercontent.com/89508906/130779613-a74c5c73-f58e-446a-b6b3-3361f6b2aaed.mp4

**Comment by Latha2564** [comment]
Logged the bug in  the wrong Repository.
